x/mock/simulation/doc.go

```diff
@@ -0,0 +1,27 @@
+/*
+Package simulation implements a simulation framework for any state machine
+built on the SDK which utilizes auth.
```

Comment:
We can actually easily remove the dependence on mock by removing the need for the `mock` module import. (Or alternatively remove the MockApp, as we no longer use that)

Reply:
Yes, that would be prudent - separate PR though